### PR TITLE
Calculate homework

### PR DESCRIPTION
**Pull request by NikczemnyOkularnik**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @NikczemnyOkularnik

**Comment by ziobron** [comment]
⏱ 3 XP extra for @NikczemnyOkularnik for on-time delivery

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+    }
+    else if(command == "subtract")
+    {
+        return std::to_string(first - second);
+    }
+    else if(command == "multiply")
+    {
+        return std::to_string(first * second);
+    }
+    else if(command == "divide")
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:19]: Add space after while, if, else etc...

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
+        }
+        else
+        {
+            return std::to_string(first / second);
+        }
+    }
+    else
+    {
+        return "Invalid data";
+    }
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:33]: If you return variable here, you don't need to use `else if` or `else`

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**